Modules/xASL_module_ASL.m

```diff
@@ -150,16 +150,45 @@
 
 %% D4. TimeEncoded parsing
```

Comment:
I would rename this D4 to its final position (check our 997 & XX merging issue).

Also, if you make this this long, I would move it to a subfunction

Reply:
It makes no sense to rename things that are later renamed in another commit. 
This only creates unnecessary conflicts. And will be renamed once we merge everything together.